vpr/src/base/region.h

```diff
@@ -49,6 +50,8 @@ class Region {
      */
     bool locked();
 
+    bool is_loc_in_reg(t_pl_loc loc);
```

Comment:
comment

Reply:
loc is assumed to be valid x, y, subtile location. Check if in legal x,y bounds of this region. If region has no subtile specified, ok, otherwise has to match